From fde4d14cb3c169490b51f9d3760a3c66e6fea2d7 Mon Sep 17 00:00:00 2001
From: 21pages <pages21@163.com>
Date: Wed, 22 May 2024 09:21:13 +0800
Subject: [PATCH] mediacodec encode remove align

Signed-off-by: 21pages <pages21@163.com>
---
 libavcodec/mediacodecenc.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/libavcodec/mediacodecenc.c b/libavcodec/mediacodecenc.c
index 984014f1b1..bd232b6a07 100644
--- a/libavcodec/mediacodecenc.c
+++ b/libavcodec/mediacodecenc.c
@@ -201,8 +201,10 @@ static av_cold int mediacodec_init(AVCodecContext *avctx)
     // Workaround the alignment requirement of mediacodec. We can't do it
     // silently for AV_PIX_FMT_MEDIACODEC.
     if (avctx->pix_fmt != AV_PIX_FMT_MEDIACODEC) {
-        s->width = FFALIGN(avctx->width, 16);
-        s->height = FFALIGN(avctx->height, 16);
+        // s->width = FFALIGN(avctx->width, 16);
+        // s->height = FFALIGN(avctx->height, 16);
+        s->width = avctx->width;
+        s->height = avctx->height;
     } else {
         s->width = avctx->width;
         s->height = avctx->height;
-- 
2.34.1

